### PR TITLE
Fix/district features

### PR DESCRIPTION
**Pull request by DylanMoriarty**
Fixes the search bar, updates the about text, & modifies the charts to resize based on how long the data is.

Grabs #6, #7

### CODE REVIEW

**Review by felskia — COMMENTED** [review]
@DylanMoriarty can you get rid of the space at the top and bottom of the charts?

File: app/assets/scripts/components/charts/horizontal-bar.js [hunk 1]
```diff
@@ -74,14 +76,15 @@ var HorizontalBarChart = React.createClass({
         right: margin.left
       });
     }
-    const innerWidth = width - margin.left - margin.right;
-    const innerHeight = height - margin.top - margin.bottom;
+    let innerWidth = width - margin.left - margin.right;
 
     // short circut if we have too small an area
-    if (innerWidth <= 0 || innerHeight <= data.length * 2) {
+    if (innerWidth <= 0) {
```
Comment (nbumbarger) [inline app/assets/scripts/components/charts/horizontal-bar.js:82]: I don't think that it's practically possible to meet this condition

**Comment by nbumbarger** [comment]
I left one inline comment, but a second issue is that the search bar doesn't function as I think it should: the only way to navigate is by typing in a governorate name, then clicking one of the entries in the autocomplete dropdown menu. The search button and enter keys don't work.
The icon should be given an `onClick` event listener that triggers the `zoomToGovernorate` function, and the search bar can be given an `onKeyPress` or `onKeyDown` event that listens for key #13 (the enter key), e.g. 
```
const handleEnter = (e) => {
  if (e.keyCode === 13)...
}
```

**Review by nbumbarger — CHANGES_REQUESTED** [review]
I left one inline comment, but a second issue is that the search bar doesn't function as I think it should: the only way to navigate is by typing in a governorate name, then clicking one of the entries in the autocomplete dropdown menu. The search button and enter keys don't work.
The icon should be given an `onClick` event listener that triggers the `zoomToGovernorate` function, and the search bar can be given an `onKeyPress` or `onKeyDown` event that listens for key #13 (the enter key), e.g. 
```
const handleEnter = (e) => {
  if (e.keyCode === 13)...
}
```

File: app/assets/scripts/components/charts/horizontal-bar.js (hunk 1 above)
Comment (DylanMoriarty) [reply app/assets/scripts/components/charts/horizontal-bar.js:82]: For some reason when you initially toggle out to the chart view, the value here goes negative from the margin values used to calculate innerWidth earlier, and it breaks the load. 

I looked into what was causing the issue, but having this condition seems to work & appears to be the solution used for Map Egypt as well.

**Pull request merged**